q2_diversity_lib/tests/data/two_feature_table.tsv

```diff
@@ -0,0 +1,4 @@
+# Constructed from biom file
+#OTU ID	S1	S2	S3	taxonomy
+O1	0.0	1.0	3.0
+O2	1.0	1.0	2.0
```

Comment:
Missing newline.

Reply:
Good catch! Looks like this is a remnant from something old and unused now. Removed.